staging/testsuite/src/test/it-resources/maven3-multinexus-project/m1/raw-pom.xml

```diff
@@ -0,0 +1,66 @@
+<!--
+
+    Sonatype Nexus (TM) Open Source Version
+    Copyright (c) 2007-2013 Sonatype, Inc.
+    All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+
+    This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+    which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+
+    Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+    of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+    Eclipse Foundation. All other trademarks are the property of their respective owners.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The test case is copied from existing [multi profile test](https://github.com/atlassian/nexus-maven-plugins/tree/master/staging/testsuite/src/test/java/org/sonatype/nexus/maven/staging/it/multiprofile). Nothing changed but use `http://127.0.0.1:port/nexus` to mimic multi nexus instances